src/main/java/frc/team2412/robot/commands/autonomous/JackFiveBallAutoCommand.java

```diff
@@ -60,54 +55,86 @@ public JackFiveBallAutoCommand(DrivebaseSubsystem drivebaseSubsystem, IntakeSubs
         // move to third ball, stop and intake for collection
         // move to where second ball was and shoot
 
-        Trajectory trajectory1 = new Trajectory(
+        public static final Trajectory PATH_1 = new Trajectory(
                 new SimplePathBuilder(new Vector2(328, 75.551), Rotation2.fromDegrees(-90))
                         .arcTo(new Vector2(293.811, 29.771), new Vector2(281.299, 92.401), Rotation2.fromDegrees(-90))
                         .lineTo(new Vector2(218.203, 60.492), Rotation2.fromDegrees(130))
                         .build(),
-                normalSpeed, 0.1);
+                NORMAL_SPEED, 0.1);
 
-        Trajectory trajectory2 = new Trajectory(
+        public static final Trajectory PATH_2 = new Trajectory(
                 new SimplePathBuilder(new Vector2(213.203, 66.492), Rotation2.fromDegrees(130))
                         .lineTo(new Vector2(195.049, 82.693), Rotation2.fromDegrees(125))
                         .build(),
-                fastSpeed, 0.1);
+                FAST_SPEED, 0.1);
 
-        Trajectory trajectory3 = new Trajectory(
+        public static final Trajectory PATH_3 = new Trajectory(
                 new SimplePathBuilder(new Vector2(195.029, 75.188), Rotation2.fromDegrees(125))
                         .lineTo(new Vector2(50.456, 85.818), Rotation2.fromDegrees(202))
                         .build(),
-                normalSpeed, 0.1);
+                NORMAL_SPEED, 0.1);
 
-        Trajectory trajectory4 = new Trajectory(
+        public static final Trajectory PATH_4 = new Trajectory(
                 new SimplePathBuilder(new Vector2(50.456, 85.818), Rotation2.fromDegrees(202))
-                        .lineTo(new Vector2(207.029, 82.188), Rotation2.fromDegrees(125))
+                        .lineTo(new Vector2(207.029, 82.188), Rotation2.fromDegrees(202))
                         .build(),
-                normalSpeed, 0.1);
+                NORMAL_SPEED, 0.1);
+
+        public static void init() {
+            System.out.println("----- 5 Ball Auto Paths Initialized -----");
+        }
+
+    }
+
+    public JackFiveBallAutoCommand(DrivebaseSubsystem drivebaseSubsystem, IntakeSubsystem intakeSubsystem,
+            IndexSubsystem indexSubsystem,
+            ShooterSubsystem shooterSubsystem, TargetLocalizer localizer) {
+
+        ShooterTargetCommand.TurretManager manager = new ShooterTargetCommand.TurretManager(shooterSubsystem,
+                localizer);
 
         addCommands(
-                new IntakeSetExtendCommand(intakeSubsystem),
-                new InstantCommand(() -> new ShooterTargetCommand(shooterSubsystem, localizer, () -> false)),
+                manager.scheduleCommand().alongWith(new IntakeSetInCommand(intakeSubsystem)),
```

Comment:
Since manager.cancelCommand is never called, this likely means the shooterTarget command scheduled by the `manager` instance here is never interrupted until `cancelAll()` in `autonomousExit`. To not be so brittle, it'd be best to cancel the command here once the turret is done being used.

Reply:
intended behavior

Reply:
the same shooter command is to be used throughout the auto